docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/duo-source.md

```diff
@@ -72,6 +72,12 @@ To configure a Duo Source:
    * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
    * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema it is ignored, known as dropped.
 
+
+:::note
+If you are using the Duo Federal edition service when connecting APIs, it's recommended to use `duofederal.com` instead of the `default duosecurity.com` domain. Our Duo C2C lets you allow to configure the API domain as it contains the specific customer ID information. For example, you can use `api-xxxx-duosecurity.com` or `api-xxxx-duofederal.com` if the Duo Federal edition service has been opted in. For more information, refer to the [Duo Federal Edition Guide](https://duo.com/docs/duo-federal-guide#duo-service-connectivity).
```

Comment:
I think the code quote is a little early and captures the word `default`. Currently the quote is `default duosecurity.com` when it should be `duosecurity.com`.